src/SIM/SIMoutput.C

```diff
@@ -83,9 +86,23 @@ bool SIMoutput::parseOutputTag (const TiXmlElement* elem)
 {
   IFEM::cout <<"  Parsing <"<< elem->Value() <<">"<< std::endl;
 
-  if (strcasecmp(elem->Value(),"resultpoints"))
+  if (strcasecmp(elem->Value(),"resultpoints") && strcasecmp(elem->Value(),"function"))
     return this->SIMinput::parseOutputTag(elem);
 
+  if (!strcasecmp(elem->Value(),"function")) {
```

Comment:
A better logic would be:
```
  if (!strcasecmp(elem->Value(),"function"))
  {
    ...do the new function stuff...
  }
  else if (strcasecmp(elem->Value(),"resultpoints"))
    return this->SIMinput::parseOutputTag(elem);

  ...do the rest...
```
and you don't need to check for "function" tag more than once

Reply:
okay